graphqlapi/src/main/java/io/stargate/graphql/web/resources/GraphqlResourceBase.java

```diff
@@ -341,11 +341,21 @@ protected static void executeAsync(
       ExecutionInput input, GraphQL graphql, @Suspended AsyncResponse asyncResponse) {
     graphql
         .executeAsync(input)
-        .thenApply(result -> asyncResponse.resume(result.toSpecification()))
+        .thenApply(
+            result -> {
+              StargateGraphqlContext context = (StargateGraphqlContext) input.getContext();
+              if (context.isOverloaded()) {
+                replyWithGraphqlError(
+                    Status.TOO_MANY_REQUESTS, "Database is overloaded", asyncResponse);
+                return true;
```

Comment:
what's the meaning of this `true` here? shouldn't you return `boolean` as the result of the `replyWithGraphqlError`?

Reply:
It's just to respect the return type of the lambda, we don't do anything with the returned value. This was kind of hidden previously when the lambda was just calling `asyncResponse.resume`.

I'm changing the return type to `Void` to clarify things.